CookieGardenHelper-reloaded/main.js

```diff
@@ -486,10 +486,12 @@ Game.registerMod("cookiegardenhelperreloaded",{
 		}
 	},
 	handleDying:function(plant, x, y){
-		if (this.config.autoHarvestCheckCpSMultDying && this.CpSMult() >= this.config.autoHarvestMiniCpSMultDying.value) {
-		  this.harvest(x, y);
-		} else if (this.config.autoHarvestDying && this.secondsBeforeNextTick() <= this.config.autoHarvestDyingSeconds) {
-		  this.harvest(x, y);
+		if (!plant.key == 'crumbspore' || !plant.key == 'doughshroom') {

```

Comment:
Check for the plant IDs for these two before handling the rest of the dying code.
Effective result is that CGHR will not harvest them even with the option selected, to get the bonus.